src/components/PostDetails.tsx

```diff
@@ -157,9 +121,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </button>
         )}
 
-        {loaded && !hasError && visible && (
-          <NewCommentForm onSubmit={addComment} />
-        )}
+        {loading && !err && visible && <NewCommentForm onSubmit={addComment} />}
```

Comment:
The condition for displaying the `NewCommentForm` should be when `loading` is false. Consider changing the condition to `{!loading && !err && visible && <NewCommentForm onSubmit={addComment} />}`.